test/index.js

```diff
@@ -130,6 +155,25 @@ test('isNode', function(t) {
     t.end();
 });
 
+test('Pass invalid children (\'\', null, undefined)', function (t) {
+    var testElement = crel('div');
+    var testedElement;
+    // ? should empty strings be ignored? Adding empty text nodes only bloats childNodes
+    // ? what about functions, objects and 0 as children
+    var testInput = ['', null, undefined];
```

Comment:
empty string should be allowed and should create an empty text node.

Consistent handling of types is important for understandability.

Reply:
Done